scout/adapter/mongo/variant.py

```diff
@@ -388,54 +388,85 @@ def get_causatives(self, institute_id, case_id=None):
 
         return causatives
 
-    def check_causatives(self, case_obj=None, institute_obj=None, limit_genes=None):
-        """Check if there are any variants that are previously marked causative
+    def _get_managed_variants(self, institute_id=None, category="snv", build="37"):
```

Comment:
Should not this code be in the `ManagedVariantHandler`?

Reply:
Sure, thats better! It will be easy enough to find from here, and easier to find there if one is looking for a method like that later.